app/components/CrashDiagramCard.tsx

```diff
@@ -2,63 +2,166 @@ import { useState } from "react";
       <Card.Header>Diagram</Card.Header>
       <Card.Body className="crash-header-card-body text-center">
-        {!diagramError && (
-          <Image
-            fluid
-            style={{
-              height: "100%",
-              maxWidth: "100%",
-            }}
-            src={`${CR3_DIAGRAM_BASE_URL}/${crash.record_locator}.jpeg`}
-            alt="crash diagram"
-            onError={() => {
-              console.error("Error loading CR3 diagram image");
-              setDiagramError(true);
-            }}
-          />
-        )}
-        {diagramError && crash.is_temp_record && (
-          <Alert variant="info">
-            <i className="fa fa-info-circle" /> Crash diagrams are not available
-            for user-created crash records
-          </Alert>
-        )}
-        {diagramError && !crash.is_temp_record && (
-          <Alert variant="danger">
-            <p>
-              The crash diagram is not available. Typically, this indicates
-              there was an error when processing this crash&aposs CR3 PDF.
-            </p>
-            <p>
-              For additional assistance, you can&nbsp;
-              <a
-                href="https://atd.knack.com/dts#new-service-request/?view_249_vars=%7B%22field_398%22%3A%22Bug%20Report%20%E2%80%94%20Something%20is%20not%20working%22%2C%22field_399%22%3A%22Vision%20Zero%20Editor%22%7D"
-                target="_blank"
-                rel="noopener noreferrer"
-              >
-                report a bug
-                <i className="fa fa-external-link"></i>
-              </a>
-              .
-            </p>
-          </Alert>
-        )}
+        <TransformWrapper>
+          {!diagramError && <ZoomResetControls setRotation={setRotation} />}
+          <TransformComponent>
+            <div style={{ height: "100%", overflow: "hidden" }}>
+              {!diagramError && (
+                <Image
+                  fluid
+                  style={{
+                    height: "100%",
+                    width: "100%",
+                    transform: `rotate(${rotation}deg)`,
+                    objectFit: "contain",
+                  }}
+                  src={`${CR3_DIAGRAM_BASE_URL}/${crash.record_locator}.jpeg`}
+                  alt="crash diagram"
+                  onError={() => {
+                    console.error("Error loading CR3 diagram image");
+                    setDiagramError(true);
+                  }}
+                />
+              )}
+              {diagramError && crash.is_temp_record && (
+                <Alert variant="info" style={{ marginTop: "50px" }}>
+                  <i className="fa fa-info-circle" />
+                  Crash diagrams are not available for temporary crash records
+                </Alert>
+              )}
+              {diagramError && !crash.is_temp_record && (
+                <Alert variant="danger" style={{ marginTop: "20px" }}>
+                  <p>The crash diagram is not available.</p>
```

Comment:
I shortened this language as well. The extra sentence .. 

>Typically, this indicates there was an error when processing this crash&aposs CR3 PDF.

.. is unusual because it is speculating on an error instead of reporting it.

Reply:
I'm with you—shorter is sweeter. One thing I was trying to convey with the old message is that we should have 100% CR3 PDF coverage for all crashes except temp crashes or crashes that occurred 10+ years ago. And otherwise we kind of do want a user to let us know if a PDF is missing. 

I guess we could over-engineer this and check the `crash_timestamp` to further refine these errors?

Reply:
I'm game with whatever you feel is best, but I am still in support of the short'n'sweet message. If you do want to define some edge cases, we can probably step back a little and pull this erroring out into a component or something to tighten up this chunk of JSX.